src/bdscript/getLeaderboardPosition.md

```diff
@@ -0,0 +1,38 @@
+# $getLeaderboardPosition
+Get position in leaderboard for given variable.
+
+## Syntax
+```
+$getLeaderboardPosition[Variable type;Variable name;Sort type;(User ID)]
+```
+
+### Parameters
+- `Variable type` `(Type: Enum || Flag: Required)`: The type of the variable. Variable types:
+  - `user` - [User variable](../guides/introduction/variables.md#user-variables)
+  - `globalUser` - [Global-User variable](../guides/introduction/variables.md#globalglobal-user-variables)
+- `Variable name` `(Type: String || Flag: Required)`: The variable name to generate the leaderboard for.
+- `Sort type` `(Type: Enum || Flag: Required)`: The sort type. Sort types:
+  - `asc` - Sorts the values in ascending order.
+  - `desc` - Sorts the values in descending order.
+- `User ID`: `(Type: Snowflake  || Flag: Optional)`: ID of the user from whom you want to get a position in the leaderboard.
+
+## Example
+```
+$nomention
+My position in leaderboards: $getLeaderboardPosition[user;Coins;asc]
```

Comment:
I would recommend changing it into $getLeaderboardPosition[user;Coins;desc], as it is much more typical use case